### DISCUSSION TITLE
如何实现下图这种嵌套的菜单

### DISCUSSION THREAD

**Discussion by cohhi** [hugo-fixit/FixIt#440] [Q&A] [answered]
![image](https://github.com/hugo-fixit/FixIt/assets/71576964/82f3efc6-53bf-46f6-8a56-007e591abf68)

hugo.toml中如何配置

**Comment by Lruihao** [comment, marked as answer]
https://github.com/hugo-fixit/docs/blob/main/config/_default/menus.zh-cn.toml
